### PR TITLE
fix:[CDS-43840]: standardise placeholder text for svc,env,infra,conne…

### PR DESCRIPTION
**Pull request by namanharness**
Standardise placeholder text for svc,env,infra & connector

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/10-common/CommandBlock/__tests__/__tests__/__snapshots__/CommandBlock.test.tsx.snap [hunk 1]
```diff
@@ -17,7 +17,7 @@ exports[`CommandBlock test command block 1`] = `
         class="bp3-popover-wrapper"
       >
         <span
-          class="bp3-popover-target bp3-popover-open"
```
Comment (AshwinBhatkal-Harness) [inline src/modules/10-common/CommandBlock/__tests__/__tests__/__snapshots__/CommandBlock.test.tsx.snap:20]: This looks like a flaky test

File: src/modules/35-connectors/strings/strings.en.yaml [hunk 2]
```diff
@@ -56,8 +56,7 @@ connectorAlreadyExist: Connector already exists for the cloud account
 loginToMasterAccount: and login to your master account
 roleARN: Create Cross Account Role
 ifReq: if required.
-selectConnector: Select Connector
-selectConnectors: Select Connectors
+entityPlaceholderText: '- Select -' #used at placeholder places to select Svc, Infra, Env, Connector
```
Comment (AshwinBhatkal-Harness) [inline src/modules/35-connectors/strings/strings.en.yaml:59]: Let's move this common/strings/strings.en.yaml

File: src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/AddEditServiceOverride.tsx [hunk 3]
```diff
@@ -142,9 +142,10 @@ export default function AddEditServiceOverride({
   })
 
   const handleServiceChange = (item: SelectOption): void => {
-    const serviceOverride = serviceOverrides?.find(svcOverride => svcOverride.serviceRef === item.value)
+    const selectedServiceItemValue = defaultTo(item, item.value)
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/AddEditServiceOverride.tsx:145]: need to rebase branch

Comment (namanharness) [reply src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/AddEditServiceOverride.tsx]: done

**Review by vikrant-harness — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Pull request merged**